test/components/session-timeout-warning.spec.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+const Component = require('../../components').sessionTimeoutWarning;
+const config = require('../../config/hof-defaults');
+
+describe('session timeout warning component', () => {
+  class Base {
+    configure() { }
+    locals() { }
+  }
+
+  let req;
+  let res;
+  let instance;
+  let next;
+  let resetStub;
```

Comment:
```suggestion
  let req, res, instance, next, resetStub; 
```

Reply:
Eslint complains when it is all on one line with this error : `Split 'let' declarations into multiple statements    one-var` so I have left them separated

Reply:
generally, most people like it the way you did it
> It makes it much easier to refactor your code later and not miss variable declarations and easier to read
so good choice